src/PayplugWoocommerceRequest.php

```diff
@@ -100,17 +100,6 @@ public function applepay_update_payment() {
 			$data = array( 'apple_pay' => $apple_pay );
 			$update = $payment->update($data);
 
-			$order = new \WC_Order($order_id);
```

Comment:
you don't need to update the order status?

Reply:
Here it turns out we should not change the status manually, it will be changed with process_payment() method of the PayplugResponse class (like other gateways) in order to have all logs, order notes and order details block for Apple Pay, and follow woocommerce order workflow.